config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/config/loader/PolicyAndFolderLoader.java

```diff
@@ -84,6 +84,11 @@ public Object loadSingle(String name, File entitiesFile) {
         throw new ConfigLoadException("Cannot load an individual policy");
     }
 
+    @Override
+    public Map<String, Object> load(File entitiesFile) {
+        throw new ConfigLoadException("Cannot load policies from config file");
```

Comment:
Here as well, is our exception message is wrong?

Reply:
we added this new method in EntityLoader interface for loading environment entities, So this method does not support loading policies from config folder

Reply:
Please add this to the method Javadoc in its interface.